commandlineparser/include/Parser.h

```diff
@@ -0,0 +1,23 @@
+#include <vector>
+#include <string>
+#include <algorithm>
+#include <sstream>
+#include <map>
+
+#include "Options.h"
+
+#ifndef PARSER_H
+#define PARSER_H
+
+namespace cmdParser
+{
+	class Parser 
+	{
+		std::map<std::string ,opts::Options*> command_list; 
+		public:
+		void add_options(std::string short_command ,std::string long_command,std::string short_description, std::string long_description);
+		
+    };
+};
```

Comment:
destructor missing for the class.. big memory leak as you are creating naked pointer map.. where is the deletion of this map